modules/folder/variables.tf

```diff
@@ -117,6 +117,7 @@ variable "logging_sinks" {
     disabled             = optional(bool, false)
     exclusions           = optional(map(string), {})
     filter               = string
+    iam                  = optional(bool, true)
```

Comment:
maybe rename to `create_iam_binding`?

Reply:
shall I then change it in all the modules (organization, folder and project)?

Reply:
ah, project already has it, ignore my comment then :)